src/managers/gameplay_parameters.py

```diff
@@ -0,0 +1,224 @@
+##################################################
+# Gameplay defining parameters
+# Paramerters depend on game mode and difficulty
+##################################################
+from src.misc.game_enums import Game_mode
+
+
+class Gameplay_Parameters:
+
+    def __init__(self):
+        self.params_list = {}
+
+        #
+        #
+        #
+        # Classic mode parameters
+        #
+        # display_name: display name is game introduction screen
+        # introduction: intoduction text in the game introduction screen
+        # game_mode: Enum for game mode
+        # score_multiplier: -
+        # spawn_chance: chance of spawing anything every frame
+        # silver_chance: if we have to spawn, chances of silver coin spawning
+        # gold_chance: see above
+        # bomb_chance: see above
+        # Note: all chances in fraction. 0.10 = 10% chance of spawning
+
+        self.params_list['classic_easy'] = {
+            'key':              'classic_easy',
+            'display_name':     'Classic Mode',
+            'game_over_name':   'Classic Easy Mode',
+            'introduction':     ('The classic game mode, avoid bombs and '
+                                 'collect as many coins as possible in 30 seconds.'
+                                 ),
+            'game_mode':        Game_mode.GAME,
```

Comment:
should we rename to `Game_mode.CLASSIC` for sticking to naming by game mode?

Reply:
Fixed it. changed .GAME to .CLASSIC and also changed the game_screen to classic_game_screen